tools/generate_helper_code_from_value_sets.rb

```diff
@@ -50,31 +50,50 @@
     "http://snomed.info/sct" => "VRDR.CodeSystems.SCT",
     "http://terminology.hl7.org/CodeSystem/v3-NullFlavor" => "VRDR.CodeSystems.PH_NullFlavor_HL7_V3",
     "http://terminology.hl7.org/CodeSystem/v3-MaritalStatus" => "VRDR.CodeSystems.PH_MaritalStatus_HL7_2x",
-    "urn:oid:2.16.840.1.114222.4.5.320" => "VRDR.CodeSystems.PH_PlaceOfOccurrence_ICD_10_WHO",
-    "http://hl7.org/fhir/us/vrdr/CodeSystem/PH-PHINVS-CDC" =>  "VRDR.CodeSystems.PH_PHINVS_CDC"
+    "http://hl7.org/fhir/administrative-gender" => "VRDR.CodeSystems.AdministrativeGender",
+    "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-bypass-edit-flag-cs" => "VRDR.CodeSystems.BypassEditFlag",
+    "http://terminology.hl7.org/CodeSystem/v3-EducationLevel" => "VRDR.CodeSystems.EducationLevel",
+    "http://terminology.hl7.org/CodeSystem/v2-0360" => "VRDR.CodeSystems.DegreeLicenceAndCertificate",
+    "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-pregnancy-status-cs" => "VRDR.CodeSystems.PregnancyStatus",
+    "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-missing-value-reason-cs" => "VRDR.CodeSystems.MissingValueReason",
+    "http://unitsofmeasure.org" => "VRDR.CodeSystems.UnitsOfMeasure",
+    "http://terminology.hl7.org/CodeSystem/v2-0136" => "VRDR.CodeSystems.YesNo"
 }
 valuesets = {
```

Comment:
There are some missing valuesets that you MAY want to include:
    - ValueSet/vrdr-system-reject-vs
    - ValueSet/vrdr-intentional-reject-vs
    - ValueSet/vrdr-transax-conversion-vs
    - ValueSet-vrdr-race-code-vs.json
    - ValueSet-vrdr-hispanic-origin-vs.json
    - ValueSet-vrdr-race-recode-40-vs.json